sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -141,6 +141,27 @@ void WinSocketSetBuffSize(curl_socket_t socket)
   }
 }
 #endif
+
+void static inline SetHeader(Azure::Core::Http::RawResponse& response, std::string const& header)
+{
+  return Azure::Core::Http::_detail::RawResponse::SetHeader(
+      response,
+      reinterpret_cast<uint8_t const*>(header.data()),
+      reinterpret_cast<uint8_t const*>(header.data() + header.size()));
+}
+
+static inline std::string GetHTTPMessagePreBody(Azure::Core::Http::Request const& request)
```

Comment:
Keep the previous comment
```suggestion
// Writes an HTTP request with RFC 7230 without the body (head line and headers)
// https://tools.ietf.org/html/rfc7230#section-3.1.1
static inline std::string GetHTTPMessagePreBody(Azure::Core::Http::Request const& request)
```